tests/storage/datavolume.go

```diff
@@ -138,6 +138,41 @@ var _ = SIGDescribe("DataVolume Integration", func() {
 		return info.VirtualSize
 	}
 
+	getUsableSize := func(vmi *v1.VirtualMachineInstance, dv *cdiv1.DataVolume) int64 {
+		pod, err := libpod.GetPodByVirtualMachineInstance(vmi, vmi.Namespace)
+		Expect(err).ToNot(HaveOccurred())
+
+		diskImagePath := "/var/run/kubevirt-private/vmi-disks/disk0/disk.img"
+
+		statfsOutput, err := exec.ExecuteCommandOnPod(
+			pod,
+			"compute",
+			[]string{"stat", "-c", "%b %B", diskImagePath},
+		)
+		Expect(err).ToNot(HaveOccurred())
+
+		var allocatedBlocks, blockSize int64
+		_, err = fmt.Sscanf(statfsOutput, "%d %d", &allocatedBlocks, &blockSize)
+		Expect(err).ToNot(HaveOccurred())
+
+		allocatedSize := allocatedBlocks * blockSize
+
+		statOutput, err := exec.ExecuteCommandOnPod(
```

Comment:
I think you have the stat/statfs variable names confused?

Reply:
Good catch